weave

```diff
@@ -870,12 +875,12 @@ case "$COMMAND" in
         echo $PROXY_CONTAINER
         ;;
     connect)
-        [ $# -eq 1 ] || usage
-        http_call $CONTAINER_NAME $HTTP_PORT POST /connect -d "peer=$1"
+        [ $# -gt 0 ] || usage
+        http_call $CONTAINER_NAME $HTTP_PORT POST /connect -d `peer_args $@`
```

Comment:
We don't use backquotes. They are evil. use `$(...)` instead. And shouldn't that `$@` be quoted?